### ISSUE TITLE
There is  an unexpected problem during training maybe?

### ISSUE THREAD

**Issue by 1937573836** [3dv-casia/BWformer#11]
Hello!I find the corner recall converged to 0 at about epoch 230,and inferred result showed no corner and the result  .obj files 0 bytes. Is it possible that the model gets stuck in a local optimum? I’ve encountered this issue before where the corner recall converges to zero, and now it’s happening again after rerunning.<img width="1661" height="139" alt="Image" src="https://github.com/user-attachments/assets/66432345-e430-43c4-afc3-97c02bb3f5a0" />

**Comment by PaulLiuYZ** [comment]
I haven’t encountered this problem before. It seems that loss_x and loss_y are much larger than loss_z, whereas they are usually at the same level. Are you training BWFormer on the Building3D dataset or on your own custom dataset?

**Comment by 1937573836** [comment]
I trained the model on Tallinn dataset of building3d.In the early stages of training, all evaluation metrics behave normally. However, as the number of training epochs increases, this situation occasionally occurs. My computational resources are limited, and I need to use checkpoint resuming, but I have not yet ruled out this as a potential cause.After this situation occurs, I usually shut down the training and then load the checkpoint model. In this way, it doesn’t happen again for a short period, but I cannot rule out that it might reoccur later.

**Comment by 1937573836** [comment]
Even the model of epoch 205th performs normally.

<img width="1504" height="107" alt="Image" src="https://github.com/user-attachments/assets/7df946e3-179e-41c1-b364-e9c823d0bde9" />

**Comment by PaulLiuYZ** [comment]
What GPU and batch size are you using for training? I'm not sure whether it's related. First, starting from the latest checkpoint helps in most cases. Second, maybe lowering the learning rate may help.

**Comment by 1937573836** [comment]
I used 5090D double gpus …  btachsize is set to 4 because of lack of memory
